data.proto

```diff
@@ -45,6 +45,9 @@ message ActorConfig {
   int32 num_input = 4;
   int32 num_action = 5;
   string environment_implementation = 6; // TODO Remove by retrieving info from the environment instead
+  string env_type = 7;
+  string env_name = 8;
```

Comment:
`env_type` and `env_name` are now inside the `environment_implementation` (also it would be nice to stop using it, I'd like to get rid of it in the ActorConfig)

Reply:
It's needed for the web client. Removing it is would be a breaking change until we figure out a better solution.

Reply:
so env_type and env_name are not needed anymore for sure. And after some thinking and some testing I think we don't have any way to cleanly remove the information about the environment specs from the actor. I'm working on a slight simplification though.

Reply:
Already resolved in `main` branch, no longer relevant